DTS-STN/canadian-dental-care-plan
# Changing letter PDF in preparation for demo

### Description
Continuation from the request partially addressed in https://github.com/DTS-STN/canadian-dental-care-plan/pull/918.
Mock letter PDF content has been changed.

### Screenshots (if applicable)
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/55502055/ece1e125-7751-4d9d-aa53-7b22f18221e5)

### Checklist

### Test Instructions
Run application and navigate to `/en/letters`. Click on the only letter that shows. The PDF should have a client number that matches the screenshot above.
